fendermint/vm/interpreter/src/fvm/interpreter.rs

```diff
@@ -57,68 +49,58 @@ struct Actor {
 
 /// Interprets messages as received from the ABCI layer
 #[derive(Clone)]
-pub struct FvmMessagesInterpreter<DB>
+pub struct FvmMessagesInterpreter<DB, M>
 where
     DB: Blockstore + Clone + Send + Sync + 'static,
+    M: ModuleBundle,
```

Comment:
Perhaps, this constraint could be relaxed to specific module traits rather than the whole bundle (also in trait implementations).